exp-player/addon/components/exp-video-preview/component.js

```diff
@@ -63,7 +62,7 @@ export default ExpFrameBaseComponent.extend(MediaReload, VideoRecord, {
         return this.get('videos')[this.get('videoIndex')];
     }),
     makeTimeEvent(eventName, extra) {
-        return this._super(`exp-physics:${eventName}`, extra);
+        return this._super(`exp-video-preview:${eventName}`, extra);
```

Comment:
I think this was actually correctly backwards-compatible, I just went ahead and changed the name since I'm fine with dealing with slightly-varying event names on the preview frame for the studies that have already started collecting data.